### DISCUSSION TITLE
[Feature Request] List Branches API should also include a default_branch field

### DISCUSSION THREAD

**Discussion by ggedde** [community/community#40863] [Apps, API and Webhooks]
### Select Topic Area

Product Feedback

### Body

Currently there is no way to know what the default branch is in the [List Branches API](https://docs.github.com/en/rest/branches/branches?apiVersion=2022-11-28#list-branches). You can get that info from the Repo API, but that means an additional api call. Instead I would also suggest adding a "default_branch" field in the List Branches API response. It would just be a boolean.

Thanks

**Comment by ossanna16** [comment]
Hi there @ggedde and welcome to our community! Thank you for the feedback!

**Reply by byrneh** [reply]
The graphql api will be able to return default and all branch details in a single call
